src/models/heat_index.js

```diff
@@ -3,6 +3,10 @@ import { round } from "../utilities/utilities";
 /**
  * Calculates the Heat Index (HI). It combines air temperature and relative humidity to determine an apparent temperature.
  *
+ * @public
+ * @memberof models
+ * @docname Heat Index
```

Comment:
I added functionality in our theme to support this custom tag and the text after it will be used for the display name of the function (the title in the docs, we can still see the real function name in the docs)

Reply:
Sick, sounds good